Nightwatch/Configuration.fs

```diff
@@ -0,0 +1,77 @@
+module Nightwatch.Configuration
+
+open System
+open System.IO
+
+open YamlDotNet.Serialization
+open YamlDotNet.Serialization.NamingConventions
+
+open Nightwatch.FileSystem
+open Nightwatch.Resources
+
+[<CLIMutable>]
+type ResourceDescription =
```

Comment:
Make this `private`?

Reply:
Yes, please.

Reply:
Unfortunately, that's not possible: YamlDotNet wants a public type; otherwise it cannot find type's constructor.

Reply:
What about `[<RequireQualifiedAccess>]`?

Reply:
I don't think it works the way we need. Look: it should be applied to **container** (e.g. module or enum), and the container's contents won't be visible without qualification.

To achieve something with that I'd need to apply that attribute to the whole module, and I don't want this.